package.json

```diff
@@ -1,43 +1,45 @@
 {
-    "name": "nt-app",
-    "version": "0.9.39",
-    "author": "Soler91",
-    "description": "Allows you to play noita with friends...",
-    "private": true,
-    "scripts": {
-        "serve": "vue-cli-service serve",
-        "build": "vue-cli-service build",
-        "lint": "vue-cli-service lint",
-        "electron:build": "vue-cli-service electron:build",
-        "electron:serve": "vue-cli-service electron:serve",
-        "postinstall": "electron-builder install-app-deps",
-        "postuninstall": "electron-builder install-app-deps"
-    },
-    "main": "background.js",
-    "dependencies": {
-        "@fortawesome/fontawesome-free": "^5.15.1",
-        "@popperjs/core": "^2.6.0",
-        "electron-updater": "^4.3.5",
-        "got": "^11.8.1",
-        "keytar": "^7.2.0",
-        "protobufjs": "^6.10.2",
-        "uuid": "^8.3.2",
-        "validator": "^13.5.1",
-        "vue": "^2.6.11",
-        "vue-form-json-schema": "^2.9.2",
-        "vue-router": "^3.2.0",
-        "vuex": "^3.6.0",
-        "ws": "^7.4.2"
-    },
-    "devDependencies": {
-        "@vue/cli-plugin-eslint": "~4.5.0",
-        "@vue/cli-plugin-router": "~4.5.0",
-        "@vue/cli-service": "~4.5.0",
-        "electron": "^9.0.0",
-        "electron-devtools-installer": "^3.1.0",
-        "eslint": "^6.7.2",
-        "eslint-plugin-vue": "^6.2.2",
-        "vue-cli-plugin-electron-builder": "~2.0.0-rc.5",
-        "vue-template-compiler": "^2.6.11"
-    }
-}
\ No newline at end of file
+  "name": "nt-app",
+  "version": "0.10.0",
+  "author": "Soler91",
+  "description": "Allows you to play noita with friends.",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc --noEmit && vite build && electron-builder",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --ignore-path .gitignore"
+  },
+  "main": "dist/electron/main/index.js",
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.1.1",
+    "electron-updater": "5.0.5",
+    "got": "^12.1.0",
+    "keytar": "^7.9.0",
+    "pinia": "^2.0.16",
+    "protobufjs": "^7.0.0",
+    "uuid": "^8.3.2",
+    "vue": "^3.2.37",
+    "vue-router": "^4.1.2",
+    "vue3-popper": "^1.5.0",
```

Comment:
Look, there's a library that does the popup/tooltip stuff. Saved me some trouble with converting the `$refs` stuff...